packages/backend/src/wallet/wallet.service.ts

```diff
@@ -76,3 +83,52 @@ export const createWallet = async (
       .json({ message: 'Unable to create wallet', success: false })
   }
 }
+
+export const verifyIdentity = async (
+  req: Request,
+  res: Response<BaseResponse>
+) => {
+  try {
+    const { email } = req.user as User
+
+    const {
+      document_type,
+      front_side_image,
+      front_side_image_mime_type,
+      face_image,
+      face_image_mime_type
+    } = await zParse(kycSchema, req)
+
+    const user = await User.query().where('email', email).first()
```

Comment:
same here, maybe going for a find by id would be better performance wise because of the index